handlers/refunds.go

```diff
@@ -0,0 +1,69 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/payments.api.ch.gov.uk/mappers"
+	"github.com/companieshouse/payments.api.ch.gov.uk/models"
+	"github.com/companieshouse/payments.api.ch.gov.uk/service"
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+// HandleCreateRefund initiates a refund from the external provider
+func HandleCreateRefund(w http.ResponseWriter, req *http.Request) {
+	if req.Body == nil {
+		log.ErrorR(req, fmt.Errorf("request body empty"))
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	vars := mux.Vars(req)
+	id := vars["paymentId"]
```

Comment:
`vars` is much too generic. Also can't this be done on one line i.e. `id = mux.Vars(req)["paymentId"]`